taworks/taform/templates/taform/ranking_status.html

```diff
@@ -12,14 +18,22 @@ <h1>Ranking Status</h1>
 
 <form action="{% url 'ranking_status' %}" method="post" enctype="multipart/form-data">
 {% csrf_token %}
-    <input type="submit" class="submitButton" name="Upload" value="Send Out Ranking Links"/>
+	<div id="email-confirm" title="Confirm Action">
+	<p class="emailButtonPopupText">Are you sure you would like to send out ranking links to all instructors? If you meant to email only one instructor, you may copy the ranking link from the table and email them manually.</p>
+	</div>
+    	<input class="emailButtonPopup" type="submit" name="Upload" value="Send Out Ranking Links"/>
```

Comment:
Can it be purple :)

Reply:
donee

Reply:
ah i see you explicitly mentioned it here.